### PR TITLE
app.py for easy terminal use + readme

### PR DESCRIPTION
**Pull request by DILLON9999**

### CODE REVIEW

**Review by cameronmoran23 — APPROVED** [review]
Looks good

**Comment by DILLON9999** [comment]
[Create easy-to-use driver application to use sample model](https://trello.com/c/Pr7Z1uq2/25-create-easy-to-use-driver-application-to-use-sample-model)

**Comment by DILLON9999** [comment]
[Create easy-to-use driver application to use sample model](https://trello.com/c/Pr7Z1uq2/25-create-easy-to-use-driver-application-to-use-sample-model)

File: PMI_Model/app.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# Get input coordinates from the user
+def get_valid_coordinates():
+    while True:
+        try:
+            lat = float(input("Enter latitude: "))
+            long = float(input("Enter longitude: "))
+            if -90 <= lat <= 90 and -180 <= long <= 180:
+                return lat, long
+            else:
+                print("Invalid coordinates")
```
Comment (cameronmoran23) [inline PMI_Model/app.py:13]: Good error handling

Comment (cameronmoran23) [inline PMI_Model/app.py:13]: Good error handling.

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# COMP-49X-Benchmark-Labs
\ No newline at end of file
+# COMP-49X-Benchmark-Labs
```
Comment (DILLON9999) [inline README.md:1]: README is easy to follow and provides good instructions.

File: PMI_Model/app.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    # Get coordinates from the user
+    lat, long = get_valid_coordinates()
+    
+    # Get date range from user
+    print("\nEnter the date range for PMI calculation:")
+    start_date = get_valid_date("Start date")
+    end_date = get_valid_date("End date")
+    
+    # Ensure end date is after start date
+    if end_date <= start_date:
```
Comment (DILLON9999) [inline PMI_Model/app.py:40]: Good error handling for edge cases

File: PMI_Model/app.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            long = float(input("Enter longitude: "))
+            if -90 <= lat <= 90 and -180 <= long <= 180:
+                return lat, long
+            else:
+                print("Invalid coordinates")
+        except ValueError:
+            print("Please enter valid numbers for coordinates.")
+
+# Get input start and end dates from the user
+def get_valid_date(prompt):
```
Comment (cameronmoran23) [inline PMI_Model/app.py:18]: Add short description/comment about "prompt" parameter.

**Pull request merged**